### ISSUE TITLE
Unstable TTS

### ISSUE THREAD

**Issue by amnipp** [amnipp/NamazuKingdomBot#12]
For some god unknown reason the TTS (probably the audio stream) doesn't work with the sentence

> Finally. Rain. We had a rough dry period due to all the heat.

I don't know why, tried several variations of it and it worked; tested it after and it worked. Something is unstable and I have no clue what it could be. It makes zero sense to me. Not sure why my crappy JS version works flawlessly but this .net version is unstable.

**Comment by amnipp** [comment]
Might be an issue with the buffer overflowing?

**Comment by amnipp** [comment]
Okay I think the issue is with NAudio. Seems that it can only do a certain amount of time; will probably split up the sentences and add to queues to play. Either that or find a better way to get the tts

**Comment by amnipp** [comment]
Figured it out, NAudio's MediaFoundationReader seems to become unstable if it is given a GET request. I believe too much data from the content stream causes it to bug out and wipes the data. To fix this I simply created the function AudioService.CreateResamplerFromStream, it uses HttpClient to get a response stream and then passes it to NAudio's Mp3FileReader. This _seems_ to be stable from the limited testing I've done. 

I have no real familiarity with NAudio so I am not even sure Mp3FileReader will be sufficient. Closing for now however I might need to reopen after some live testing.
